fix #34068 hochu.tv

zloyden opened the pull request:
https://github.com/AdguardTeam/AdguardFilters/issues/34068
Branding 1xbet not reproduced but here I removed redirect from player when you click on play/pause. Also blocked push-notifications.

zloyden committed:
fix #34068 hochu.tv

BlazDT commented on AnnoyancesFilter/sections/push-notifications.txt:
```diff
@@ -19,6 +19,7 @@ wallstreet-online.de,dni.ru,polit.info#%#if (window.PushManager) { window.PushMa
 ||pushtora.com^$third-party
 !
 ||install.searchfrit.com^$third-party
+||selectroduced.com^$third-party
```
It is a fake push notification. Should be moved to adservers.

BlazDT commented on RussianFilter/sections/general_extensions.txt:
```diff
@@ -20,6 +20,9 @@ s4galaxy.ru,3dnews.ru,lenovo-smart.ru#%#Object.defineProperty(window, 'SocialMar
 windowspro.ru#%#AG_abortInlineScript(/browserDetect/, 'document.createElement');
+! https://github.com/AdguardTeam/AdguardFilters/issues/34068
+cdntvnet.com#%#//scriptlet("prevent-window-open")
+cdntvnet.com#%#window.open = function() { };
```
Was not working for me.

Should work though:
||cdntvnet.com/jquerymin18.js

BlazDT reviewed (changes requested):
||youtvonlinefree.com/foradu^

Stealth referrer on http://hochu.tv/fap-tv-compilation.html:
@@||cdntvnet.com/fap-tv-compilation.php$stealth

zloyden committed:
fix #34068

zloyden commented:
Added `||youtvonlinefree.com/foradu/`.
For stealth-mode referrer I changed a bit rule because old version worked only on one category.
> Was not working for me.

That's strange. I checked these rules on extension and app - works fine on my side, but I change to your variant. `||cdntvnet.com/jquerymin18.js` is enough for block redirect.
